plugin-configuration.md - exampel with "..." in extending config (#790)
parcel-bundler/website: src/configuration/plugin-configuration.md

{% endsamplefile %}
{% endsample %}

> If default config need to be extended (for example additional reporters needs to be added, but default ones need to stay), add `"..."` to extend list provided by config specified in `extends` field.

Reviewer: ```suggestion
To add additional plugins while not overwriting the but default ones, add `"..."` to use the plugins provided by config specified in the `extends` field. For example to add an additional reporter:
```

Author: Oh, that's sounds better, thanks!

{% sample %}
{% samplefile ".parcelrc" %}

```json/1
{
  "extends": "@parcel/config-default",

>   "reporters": ["...", "@parcel/reporter-dev-server"]

Reviewer: The dev server is already enabled by default...
```suggestion
  "reporters": ["...", "parcel-reporter-custom"]
```

Author: :+1: Yep, also `parcel-reporter-custom` seems better.
